### PR TITLE
Product rebranding rebranding: Microsoft Defender for Cloud and Microsoft Container Registry 

### PR DESCRIPTION
**Pull request by lanicolas**
Rename Azure Security Center for Microsoft Defender and Microsoft Container Registry on all scenarios where it is referenced.

### CODE REVIEW

**Comment by lanicolas** [comment]
All images are now updated to reflect Defender instead of ASC

File: azure_arc_k8s_jumpstart/cluster_api/capi_defender_extension/azure_defender_k8s_extension.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ export password='<Your Azure service principal password>'
 export tenantId='<Your Azure tenant ID>'
 export resourceGroup='<Azure resource group name>'
 export arcClusterName='<Azure Arc Cluster Name>'
-export k8sExtensionName='<Azure Defender Extension Name' #default: 'microsoft.azuredefender.kubernetes'
+export k8sExtensionName='<Azure Defender Extension Name>' #default: 'microsoft.azuredefender.kubernetes'
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/cluster_api/capi_defender_extension/azure_defender_k8s_extension.sh:11]:
```suggestion
export k8sExtensionName='<Microsoft Defender for Cloud extension name>' #default: 'microsoft.azuredefender.kubernetes'
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+linkTitle: "Integrate Microsoft Defender for Cloud with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
 weight: 1
 description: >
 ---
 
-## Integrate Azure Defender with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions
+## Integrate Microsoft Defender for Cloud with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions
 
-The following README will guide you on how to enable [Azure Defender](https://docs.microsoft.com/en-us/azure/security-center/defender-for-kubernetes-introduction) for a Cluster API that is projected as an Azure Arc connected cluster.
+The following README will guide you on how to enable [Microsoft Defender for Cloud](https://docs.microsoft.com/en-us/azure/defender-for-cloud/defender-for-containers-introduction?tabs=defender-for-container-arch-aks) for a Cluster API that is projected as an Azure Arc connected cluster.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md:11]:
```suggestion
The following README will guide you on how to enable [Microsoft Defender for Cloud](https://docs.microsoft.com/en-us/azure/defender-for-cloud/defender-for-containers-introduction?tabs=defender-for-container-arch-aks) for a Cluster API that is projected as an Azure Arc-connected cluster.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md [hunk 3]
```diff
@@ -75,23 +75,23 @@ For you to get familiar with the automation and deployment flow, below is an exp
...
+* User is running the shell script. The script will use the extension management feature of Azure Arc to deploy the Microsoft Defender for Cloud extension on the Azure Arc connected cluster.
 
-* User is veryfing the cluster is shown in Azure Defender and that the extension is deployed.
+* User is veryfing the cluster is shown in Microsoft Defender for Cloud and that the extension is deployed.
 
-* User is simulating a security alert on the Kubernetes cluster that will cause Azure Defender to detect this activity and trigger a security alert.
+* User is simulating a security alert on the Kubernetes cluster that will cause Microsoft Defender for Cloud to detect this activity and trigger a security alert.
 
-## Create Azure Defender extensions instance
+## Create Defender extensions instance
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md:84]:
```suggestion
## Create Microsoft Defender for Containers extensions instance
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md [hunk 4]
```diff
@@ -75,23 +75,23 @@ For you to get familiar with the automation and deployment flow, below is an exp
...
 
-To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Azure Defender extension on your Azure Arc-enabled Kubernetes cluster.
+To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Microsoft Defender for Cloud extension on your Azure Arc-enabled Kubernetes cluster.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md:86]:
```suggestion
To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Microsoft Defender for Containers extension on your Azure Arc-enabled Kubernetes cluster.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md [hunk 5]
```diff
@@ -75,23 +75,23 @@ For you to get familiar with the automation and deployment flow, below is an exp
...
 
-* Before integrating the cluster with Azure Defender, click on the "Security (preview)" blade for the connected Arc cluster to show how the cluster is not currently being assessed by Azure Defender.
+* Before integrating the cluster with Microsoft Defender for Cloud, click on the "Security (preview)" blade for the connected Arc cluster to show how the cluster is not currently being assessed by Microsoft Defender for Cloud.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md:88]:
```suggestion
* Before integrating the cluster with Microsoft Defender for Cloud, click on the "Security (preview)" blade for the connected Azure Arc cluster to show how the cluster is not currently being assessed by Microsoft Defender for Cloud.
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md [hunk 6]
```diff
@@ -101,13 +101,13 @@ To create a new extension Instance, we will use the _k8s-extension create_ comma
...
-  * Create Azure Defender k8s extension instance
+  * Create Defender k8s extension instance
 
-* You can see that Azure Defender is enabled once you visit the security tab section of the Azure Arc-enabled Kubernetes cluster resource in Azure.
+* You can see that Microsoft Defender for Cloud is enabled once you visit the security tab section of the Azure Arc-enabled Kubernetes cluster resource in Azure.
 
 ![Screenshot extension deployment security tab](./04.png)
 
-* Also verify under the Extensions section of the Azure Arc-enabled Kubernetes cluster that the Azure Defender extension is correctly installed.
+* Also verify under the Extensions section of the Azure Arc-enabled Kubernetes cluster that the Defender extension is correctly installed.
```
Comment (likamrat) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_defender_extension/_index.md:110]:
```suggestion
* Also verify under the Extensions section of the Azure Arc-enabled Kubernetes cluster that the Microsoft Defender extension is correctly installed.
```

**Review by likamrat — CHANGES_REQUESTED** [review]
Minor updates

**Comment by lanicolas** [comment]
changed

**Review by likamrat — APPROVED** [review]
LGTM

**Pull request merged**